dgp/gui/workspaces/PlotTab.py

```diff
@@ -79,37 +89,45 @@ def _setup_ui(self):
         qhbl_main.addWidget(dock_widget)
         self.setLayout(qhbl_main)
 
-    def _channel_added(self, plot: int, item: QStandardItem):
-        self.plot.add_series(item.data(Qt.UserRole), plot)
+    def _channel_added(self, row: int, item: QStandardItem):
+        series: pd.Series = item.data(Qt.UserRole)
+        if series.max(skipna=True) < 1000:
+            axis = Axis.RIGHT
+        else:
+            axis = Axis.LEFT
+        self._plot.add_series(item.data(Qt.UserRole), row, axis=axis)
+        # plot = self._plot.get_plot(row=row)
+        # plot.autoRange(items=plot.curves)
 
     def _channel_removed(self, item: QStandardItem):
-        self.plot.remove_series(item.data(Qt.UserRole))
+        # TODO: Fix this for new API
+        series: pd.Series = item.data(Qt.UserRole)
+        indexes = self._plot.find_series(series.name)
+        for index in indexes:
+            self._plot.remove_series(*index)
 
     def _clear_plot(self):
-        self.plot.clear()
+        self._plot.clear()
 
     def _toggle_selection(self, state: bool):
-        self.plot.selection_mode = state
+        self._plot.selection_mode = state
```

Comment:
This throws an `AttributeError` because there is no setter selection_mode in `LineSelectPlot`. I suppose you meant to use `set_select_mode` here.  I found this when trying to toggle into selection mode in the UI, but I see that there is a unit test for this method and it uses `set_select_mode`.